swagger-jsdoc/swagger-jsdoc-tests.ts

```diff
@@ -0,0 +1,23 @@
+/// <reference path="swagger-jsdoc.d.ts" />
+/// <reference path="../express/express.d.ts" />
+
+import * as express from 'express';
+const swaggerJsdoc = require('./swagger-jsdoc');
```

Comment:
`require` is a function returning `any`. Please use `import swaggerJsdoc = require("./swagger-jsdoc");` instead.